src/components/GradesView/GradebookTable/messages.js

```diff
@@ -1,6 +1,11 @@
 import { defineMessages } from '@edx/frontend-platform/i18n';
 
 const messages = defineMessages({
+  fullNameHeading: {
+    id: 'gradebook.GradesView.table.headings.fullName',
+    defaultMessage: 'Full Name*',
```

Comment:
I know this is a pattern we use other places but it is a bad pattern, putting the asterisk styling in the message itself. It should be added as a separate span or other styling.

```suggestion
    defaultMessage: 'Full Name',
```